pkg/api/roles.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+// API related actions
+const (
+	ActionProvisioningReload = "provisioning:reload"
+)
+
+// API related scopes
+const (
+	ScopeServicesAll           = "services:*"
+	ScopeServicesDashboards    = "services:dashboards"
```

Comment:
I wonder if the pattern is correct, since on the contrary to access control, are they really services?

Reply:
Maybe this would be better:
```go
ScopeProvisionersDashboards = "provisioners:dashboards"
```

Reply:
I agree that services is confusing. I like your suggestion about using `provisioners`. I was thinking about `components`, but `provisioners` aligns more with what we are calling those.

Reply:
Done :)